rfcs/reftest_variants.md

```diff
@@ -0,0 +1,38 @@
+# RFC 71: Support adding URI fragment for reftests
+
+## wpt issue
+
+https://github.com/web-platform-tests/wpt/issues/26364
+
+## Summary
+
+Adding URI query and fragment parts are allowed for testharness.js tests via a
+`<meta name="variant">` in the test file. This is a request to add support for
+the same meta element for reftests. Text fragment anchors are only allowed as
+top level navigations and requires support for adding a fragment to the test
+url.
+
+## Details
+
+A text fragment url for highlighting text in a page can be done with fragments
```

Comment:
Is text-fragments the only motivating use case here? It would be a stronger proposal with more use cases than just a WCIG spec with (afaict) limited vendor interest.

Reply:
> Is text-fragments the only motivating use case here? It would be a stronger proposal with more use cases than just a WCIG spec with (afaict) limited vendor interest.

The highlight styling is in a CSSWG draft which is what I was writing tests for:

https://drafts.csswg.org/css-pseudo-4/#selectordef-target-text

I don't know about any other use cases atm, assuming the meta support wouldn't be more convenient than:

`<script>window.location.hash = "#target";</script>`

for other fragments.

Reply:
I think one could set `window.location.hash` for other fragments, but that's not really intuitive (e.g. subtleties like screenshot timing -- Does this trick delay `load`? / Is `reftest-wait` needed?), and the semantics are not exactly the same.

The proposed implementation here seems pretty straightforward and is a natural generalization of the existing `variants` system, so I think the convenience/intuitiveness alone is well worth the effort, let alone the specific use case unblocked by it.